src/guide/class-and-style.md

```diff
@@ -3,6 +3,7 @@
 A common need for data binding is manipulating an element's class list and its inline styles. Since they are both attributes, we can use `v-bind` to handle them: we only need to calculate a final string with our expressions. However, meddling with string concatenation is annoying and error-prone. For this reason, Vue provides special enhancements when `v-bind` is used with `class` and `style`. In addition to strings, the expressions can also evaluate to objects or arrays.
 
 ## Binding HTML Classes
+<VideoLesson href="https://vueschool.io/lessons/dynamic-css-classes-with-vue-3?friend=vuejs" title="Free Vue.js Dynamic Classes Lesson">Watch a free video lesson on Vue School</VideoLesson>
```

Comment:
At 1:05, when demoing the object syntax, what do you think about adding in the following audio? 
"The conditional provided as the value can be anything truthy or falsey. It can be an data property as shown here but it could also be a call to a method, a computed prop, or a ternary statement." Thanks!

Reply:
I'm not sure how that would help.

At around the 2:00 mark some code is added like this:

```
:class="[item.purchased ? 'strikeout' : '']"
```

The array is not required. The 'correct' way to write that conditional would be:

```
:class="item.purchased ? 'strikeout' : ''"
```

The role of the array is purely as a combinator, allowing multiple strings and objects to be combined. The video presents arrays as though they're a required part of adding a conditional class via an expression but the two are largely orthogonal.

Reply:
I see what you're saying. Thinking on this.

Reply:
At 00:23 I've replaced "Vue has 2 ways for adding dynamic classes" 
with "Vue has 2 ways for adding dynamic classes in addition to the normal attribute binding. Those 2 ways are the array and object syntax."
Also around 2:46 I right after demonstrating the 
```
:class="[item.purchased ? 'strikeout' : '']"
```
I've added "It’s worth noting that I’m only using the array syntax here in order quickly move to multiple expressions in just a minute. If you only need one expression you can omit the array brackets all together and it works just like any other attribute binding."

Reply:
I think this explanation is still problematic.

At about 1:49 it says it's going to demonstrate the array syntax. From then until 2:45, almost everything that gets said in the narration is about using a ternary expression, not the array syntax. e.g. 'It [the array syntax] allows adding multiple classes to one condition'. Explaining the use of ternaries is really valuable but I don't think it should presented as part of the array syntax. The array syntax itself isn't really explained during that section.

I think it would be better to discuss the ternary approach before showing objects or arrays. The array syntax can then be presented just as a way to combine multiple expressions.